pkgs/by-name/ln/lngen/package.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, fetchFromGitHub
+, haskellPackages
+, lib
+}:
+with haskellPackages;
+mkDerivation {
+  pname = "lngen";
+  version = "unstable-2023-10-17";
+  src = fetchFromGitHub {
+    owner = "plclub";
+    repo = "lngen";
+    rev = "c7645001404e0e2fec2c56f128e30079b5b3fac6";
+    sha256 = "2vUYHtl9yAadwdTtsjTI0klP+nRSYGXVpaSwD9EBTTI=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.